examples/consistency_distillation/train_lcm_distill_sd_wds.py

```diff
@@ -1097,7 +1097,7 @@ def compute_embeddings(prompt_batch, proportion_empty_prompts, text_encoder, tok
     for epoch in range(first_epoch, args.num_train_epochs):
         for step, batch in enumerate(train_dataloader):
             with accelerator.accumulate(unet):
-                image, text, _, _ = batch
+                image, text = batch
```

Comment:
Just to confirm, is this change because the SD distillation script currently has a bug where it assumes that the `Text2ImageDataset.train_dataloader` batch consists of `(image, text, orig_size, crop_coords)` like the SDXL dataloader instead of just `(image, text)`:

https://github.com/huggingface/diffusers/blob/110ac7f060a494d03a5f1ce4acad4b3bbb280952/examples/consistency_distillation/train_lcm_distill_sd_wds.py#L176-L178

and is otherwise unrelated to CC12M support?

Reply:
Looks like [711e468](https://github.com/huggingface/diffusers/pull/5908/commits/711e468474f2080bb0e5f4ce3a984430550147cc) mentions this. Feel free to close :).

Reply:
Yes, exactly, it's actually a bug and not directly related to CC12M :)